src/manifest.json

```diff
@@ -39,21 +39,41 @@
         "mac": "MacCtrl+Period"
       },
       "description": "Open containers panel"
+    },
+    "open_container_0": {
+      "suggested_key": {
+        "default": "Ctrl+Shift+0"
```

Comment:
It will be confusing in code, but to open a tab in the "first" container, should the key be `1` instead?